### PR TITLE
Share more code between STRIP formats

### PR DESCRIPTION
**Pull request by kholia**

### CODE REVIEW

File: src/strip_common.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	unsigned char data[1024];
+};
+
+extern struct fmt_tests strip_tests[];
+
+int strip_valid(char *ciphertext, struct fmt_main *self);
+
+void *strip_get_salt(char *ciphertext);
+
+int strip_verify_page(unsigned char *page1);
```
Comment (magnumripper) [inline src/strip_common.h]: Functions are also `extern`

Comment (magnumripper) [reply src/strip_common.h]: Apparently we have 80+ files where `extern` is missing. Not sure if it ever matters, I think not?

Comment (kholia) [reply src/strip_common.h]: Probably not, I think.

Comment (magnumripper) [reply src/strip_common.h]: Apparently `extern` is the default. I wasn't quite sure there was a difference or not, but there isn't.

Comment (kholia) [reply src/strip_common.h]: All the compilers have loved my code so far (just kidding!) ;)

**Pull request merged**